tests/Functional/Schema/MySQL/ComparatorTest.php

```diff
@@ -214,6 +215,22 @@ public function testMariaDb1043NativeJsonUpgradeDetected(): void
         ComparatorTestUtils::assertDiffNotEmpty($this->connection, $this->comparator, $table);
     }
 
+    public function testEnumDiffDetected(): void
+    {
+        $table = new Table('enum_test_table');
+
+        $table->addColumn('enum_col', Types::ENUM, ['members' => ['a', 'b']]);
+        $this->dropAndCreateTable($table);
+
+        ComparatorTestUtils::assertNoDiffDetected($this->connection, $this->comparator, $table);
+
+        // Alter column to previous state and check diff
+        $sql = 'ALTER TABLE enum_test_table CHANGE enum_col enum_col ENUM(\'NOT_A_MEMBER_ANYMORE\') NOT NULL';
+        $this->connection->executeStatement($sql);
+
+        ComparatorTestUtils::assertDiffNotEmpty($this->connection, $this->comparator, $table);
```

Comment:
It's good that the diff is not empty, but we should actually have an opinion on what is in that diff, shouldn't we.